example/pyo3/run.py

```diff
@@ -53,21 +53,30 @@ def parse_args() -> Tuple[AccelerationMode, Path, str, Path]:
         help='モード ("AUTO", "CPU", "GPU")',
     )
     argparser.add_argument(
-        "open_jtalk_dict_dir",
+        "--dict",
```

Comment:
pathなのか何なのか分かりづらいのでたぶん`dict_dir`のが良いかも？
```suggestion
        "--dict_dir",
```